plugins/by-name/dap-python/default.nix

```diff
@@ -0,0 +1,94 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    optionalString
+    types
+    ;
+  inherit (lib.nixvim)
+    defaultNullOpts
+    mkNullOrOption
+    toLuaObject
+    ;
+
+  dapHelpers = import ../dap/dapHelpers.nix { inherit lib; };
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "dap-python";
+  package = "nvim-dap-python";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  extraOptions = {
+    adapterPythonPath = lib.mkOption {
+      default = lib.getExe (pkgs.python3.withPackages (ps: with ps; [ debugpy ]));
+      defaultText = lib.literalExpression ''
+        lib.getExe (pkgs.python3.withPackages (ps: with ps; [ debugpy ]))
+      '';
+      description = "Path to the python interpreter. Path must be absolute or in $PATH and needs to have the debugpy package installed.";
+      type = types.str;
+    };
+
+    customConfigurations = mkNullOrOption (types.listOf dapHelpers.configurationOption) "Custom python configurations for dap.";
+
+    resolvePython = defaultNullOpts.mkLuaFn null ''
+      Function to resolve path to python to use for program or test execution.
+      By default the `VIRTUAL_ENV` and `CONDA_PREFIX` environment variables are used if present.
+    '';
+
+    testRunner = mkNullOrOption (types.either types.str types.rawLua) ''
+      The name of test runner to use by default.
+      The default value is dynamic and depends on `pytest.ini` or `manage.py` markers.
+      If neither is found "unittest" is used.
+    '';
```

Comment:
Can we convey the "dynamic" default value using a `literalMD` pluginDefault?

Reply:
Not familiar with it enough to add extra detail. Looks like a copy pasta from the comments inside dap-python and it refers to stuff in pytest.ini

Reply:
I'm fine with keeping the wording intact. I guess I was thinking something like:

```nix
{
  description = "The name of test runner to use by default.";
  pluginDefault = literalMD ''
    The default value is dynamic and depends on `pytest.ini` or `manage.py` markers.
    If neither is found "unittest" is used.
  '';
}
```

Reply:
Ah, I see. Sure.